data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/paths/_eth_v2_validator_blocks_{slot}.json

```diff
@@ -38,15 +38,21 @@
       "200" : {
         "description" : "Request successful",
         "content" : {
-          "application/octet-stream" : {
+          "application/json" : {
             "schema" : {
-              "type" : "string",
-              "format" : "binary"
+              "description" : "Request successful",
+              "type" : "object",
+              "oneOf" : [ {
+                "$ref" : "#/components/schemas/ProduceBlockV2Response"
+              }, {
+                "$ref" : "#/components/schemas/ProduceBlockBlockContentsResponse"
```

Comment:
shouldn't it be `ProduceBlockBlockContentsResponse` ?

Reply:
You mean `ProduceBlockContentsResponse` 😁 My bad! That's a typo... Fixed it